airgun/entities/host_new.py

```diff
@@ -908,6 +908,12 @@ def get_insights(self, entity_name):
         view.wait_displayed()
         self.browser.plugin.ensure_page_safe()
         return view.insights.read()
+    
+    def get_cves(self, entity_name):
```

Comment:
This is an unnecessary method, you can use https://github.com/SatelliteQE/airgun/blob/da69b696cd14095420d947348fa044fd5ac9179a/airgun/entities/host_new.py#L56 to read this widget instead